mkosi/bubblewrap.py

```diff
@@ -80,6 +80,8 @@ def bwrap(
         "bwrap",
         "--ro-bind", "/usr", "/usr",
         "--ro-bind-try", "/nix/store", "/nix/store",
+        "--ro-bind-try", "/etc/ssl", "/etc/ssl",
```

Comment:
We already handle `/etc/ssl` in `finalize_mounts()`.

Reply:
Oh. I misread the code in `finalize_mounts`. Will fix this.